source/installation_and_configuration/opennebula_services/fireedge.rst

```diff
@@ -84,6 +84,17 @@ The FireEdge server configuration file can be found in ``/etc/one/fireedge-serve
 +-------------------------------------------+--------------------------------+----------------------------------------------------+
 | ``guacd/host``                            | ``localhost``                  | Connection hostname/IP of guacd server             |
 +-------------------------------------------+--------------------------------+----------------------------------------------------+
+| ``auth``                                  | ``opennebula``                 | Authentication driver for incoming requests:       |
+|                                           |                                | **opennebula** the authentication will be done by  |
+|                                           |                                | the opennebula core using the driver defined for   |
```

Comment:
I believe this should be in Uppercase as it is the company name

```suggestion
|                                           |                                | the OpenNebula core using the driver defined for   |
```

Reply:
done!

Reply:
This is still not done